src/components/shared/card.tsx

```diff
@@ -119,29 +136,35 @@ const anchorStyles = css`
 	${darkModeCss`
         color: ${neutral[86]};
     `}
+	display: flex;
+	flex-direction: column;
+	height: 100%;
 `;
 
 const headingWrapperStyles = css`
-	padding: ${remSpace[3]};
-	min-height: 10rem;
+	padding: 2px ${remSpace[2]} ${remSpace[4]};
```

Comment:
This padding doesn't fit onto the design system spacing chart @benwuersching? If not can we make it `0.125rem` @faresite?

Reply:
This padding is per @benwuersching comments above. I can remify it though 👍